### PR TITLE
Add attribute to event

### PR DESCRIPTION
**Pull request by feymartynov**

### CODE REVIEW

File: src/app/endpoint/event.rs [hunk 1]
```diff
@@ -392,6 +414,8 @@ mod tests {
                 "message",
                 "authors",
                 &account_id,
+                "attributes",
```
Comment (khodzha) [inline src/app/endpoint/event.rs:417]: maybe we can do with existing authz objects?

File: src/db/event.rs [hunk 2]
```diff
@@ -57,6 +58,11 @@ impl Object {
         self.label.as_ref().map(|val| val.as_ref())
     }
 
+    #[cfg(test)]
+    pub(crate) fn attribute(&self) -> Option<&str> {
+        self.attribute.as_ref().map(|val| val.as_ref())
```
Comment (khodzha) [inline src/db/event.rs:63]: i think this can be written as `self.attribute.as_deref()`

File: src/db/event.rs [hunk 3]
```diff
@@ -57,6 +58,11 @@ impl Object {
         self.label.as_ref().map(|val| val.as_ref())
```
Comment (khodzha) [inline src/db/event.rs:58]: `as_deref()` while we at it

**Review by khodzha — APPROVED** [review]

File: docs/src/api/event.md [hunk 4]
```diff
@@ -22,6 +22,7 @@ room_id              | uuid     | _required_ | The room's identifier to which th
 type                 | string   | _required_ | The event type.
 set                  | string   |       type | The set to which the event is related.
 label                | string   | _optional_ | A label to identify an element within the set.
+attribute            | string   | _optional_ | An attribute for authorization and filtering.
```
[the remark this answers is not in the document]
Comment (feymartynov) [reply docs/src/api/event.md:25]: I don't know how to explain this without getting into pinned messages example :(

**Review by manifest — APPROVED** [review]
We discussed to simplify authz object – to use attribute value in place of claim/event.

**Pull request merged**